R/reorder.R

```diff
@@ -45,15 +49,24 @@
 #'   geom_point() +
 #'   geom_line() +
 #'   labs(colour = "Chick")
-fct_reorder <- function(.f, .x, .fun = median, ..., .desc = FALSE) {
+fct_reorder <- function(.f, .x, .fun = median, ..., .na_rm = FALSE, .na_last = TRUE, .desc = FALSE) {
   f <- check_factor(.f)
   stopifnot(length(f) == length(.x))
   check_dots_used()
 
+  miss <- is.na(.x)
```

Comment:
FWIW I also did some checking to make sure that it currently works as expected if there are any missing values or missing levels in `.f`. It seems fine, so this PR is definitely just focused on `.x`.